src/ios/GoogleMaps/PluginPolyline.m

```diff
@@ -102,12 +102,12 @@ -(void)create:(CDVInvokedUrlCommand *)command
 
       // Since this plugin provide own click detection,
       // disable default clickable feature.
-      polyline.tappable = NO;
+      polyline.tappable = [[json valueForKey:@"clickable"] boolValue];
```

Comment:
This is wrong. If you set `polyline.tappable = YES`, this plugin can not get clicked latlng.

Reply:
I agree with you, but the code to try to find if the click is on the polyline don't work see my git project example in #2845